capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -46,43 +49,302 @@ Le due scatole conterranno il numero 5.
 
 Vediamo come scrivere i programmi corrispondenti a queste due istruzioni: 
 
++--------------------------+-------------------------+
+|**Programma1 PIPPO=PLUTO**| **Il risultato sarà:**  |
++==========================+=========================+
+| PIPPO=5                  |                         |
+|                          |                         |
+| PLUTO=15                 |                         |
+|                          |                         |
+| print("PIPPO=",PIPPO)    | PIPPO=5                 |
+|                          |                         |
+| print("PLUTO=",PLUTO)    | PLUTO=15                |
+|                          |                         |
+| PIPPO=PLUTO              |                         |
+|                          |                         |
+| print("Dopo l'esecuzione | Dopo l'istruzione       |
+| dell'istruzione          |                         |
+|                          | PLUTO=15                |
+| PIPPO=",PIPPO)           |                         |
+|                          |                         |
+| print("Dopo l'esecuzione | PIPPO=15                |                  
+| dell'istruzione          |                         |
+| PLUTO=",PLUTO)           |                         |         
+|                          |                         |
++--------------------------+-------------------------+
+|**Programma2 PLUTO=PIPPO**| **Il risultato sarà:**  |
```

Comment:
Qui invece potresti mettere il codice in un activecode e farlo eseguire all'utente così vede lui il risultato